src/common/tweaksManager.js

```diff
@@ -1,2 +1,2223 @@
Manager.DEFAULT_TWEAKS.isElementCompatible(e)},{match:{hostname:"wikipedia.org"},getInputAreaWrapperTweaks(e){const t=TweaksManager.DEFAULT_TWEAKS.getInputAreaWrapperTweaks(e);return{createMutationObserver:(e,a)=>t.createMutationObserver(e,a),getParsingDetector(e){const a=t.getParsingDetector(e),r={e:"ᵉ",er:"ᵉʳ",re:"ʳᵉ",ers:"ᵉʳˢ",res:"ʳᵉˢ",d:"ᵈ",de:"ᵈᵉ",des:"ᵈᵉˢ"};return{isIgnored(e){if(!["SPAN","TIME","SMALL","ABBR","A"].includes(e.tagName))return a.isIgnored(e);const t=e.getAttribute("typeof");return!(t&&["mw:DisplaySpace","mw:Entity","mw:Transclusion"].includes(t)||e.classList.contains("ve-ce-mwTransclusionNode"))&&a.isIgnored(e)},isSignature:e=>a.isSignature(e),isQuote:e=>a.isQuote(e),getReplacementText:(e,t,r)=>a.getReplacementText(e,t,r),isBlock:(e,t)=>a.isBlock(e,t),createMutationObserver:e=>a.createMutationObserver(e),replaceText:(e,t,n)=>r[e]&&"SUP"===n.tagName?r[e]:a.replaceText(e,t,n),isWhiteSpace:e=>a.isWhiteSpace(e)}},getFullTextInfo:(e,a)=>t.getFullTextInfo(e,a),getSelectedText:()=>t.getSelectedText(),getSelection:()=>t.getSelection()}}},{match:{hostname:"wikitree.com"},isElementCompatible:e=>e.classList.contains("CodeMirror-code")&&"true"===e.getAttribute("contenteditable")||TweaksManager.DEFAULT_TWEAKS.isElementCompatible(e)},{match:{hostname:["mastodon.social","mastodon.cloud","mamot.fr","mastodon.online","mstdn.jp","switter.at","pawoo.net","sinblr.com","social.tchncs.de"]},getToolbarAppearance(e){const t=!!e.closest(".columns-area__panels__pane__inner"),a=TweaksManager.DEFAULT_TWEAKS.getToolbarAppearance(e);return{isVisible:(e,t)=>a.isVisible(e,t),getPosition(e,r,n,s,o){const i=a.getPosition(e,r,n,s,o);return i&&i.top&&t?Object.assign(i,{top:parseInt(i.top)-n.getDocumentScroll().top+"px",fixed:!0}):i},getZIndex:(e,t,r)=>a.getZIndex(e,t,r),getClassName:e=>a.getClassName(e)}}},{match:{hostname:"hey.com"},getInputAreaWrapperTweaks(e){const t=TweaksManager.DEFAULT_TWEAKS.getInputAreaWrapperTweaks(e);return{createMutationObserver:(e,a)=>t.createMutationObserver(e,a),getParsingDetector(e){const a=/^\s*On\s+(\w+)\s+\d+,\s+\d+,\s+.+\s+wrote:\s*$/,r=t.getParsingDetector(e);return{isIgnored:e=>r.isIgnored(e),isSignature:e=>r.isSignature(e),isQuote:e=>r.isQuote(e),getReplacementText:(e,t,a)=>r.getReplacementText(e,t,a),isBlock:(e,t)=>r.isBlock(e,t),createMutationObserver:e=>r.createMutationObserver(e),replaceText:(e,t,n)=>(e=e.replace(a,e=>"\ufeff".repeat(e.length)),r.replaceText(e,t,n)),isWhiteSpace:e=>r.isWhiteSpace(e)}},getFullTextInfo:(e,a)=>t.getFullTextInfo(e,a),getSelectedText:()=>t.getSelectedText(),getSelection:()=>t.getSelection()}}},{match:{url:/thunderbird.net|thunderbird-mail.de|\/thunderbird/},init(){const e=StorageController.create();e.onReady(()=>{e.updateStatistics({isThunderbirdUser:!0})}),TweaksManager.DEFAULT_TWEAKS.init()}},{match:{hostname:["clubdesk.com","clubdesk.de"]},supported:()=>!1,unsupportedMessage:()=>i18nManager.getMessage("siteCannotBeSupported")}],"undefined"!=typeof module&&(module.exports=TweaksManager);
```

Comment:
I reformatted this so I would be able to more easily navigate what to delete
Should I revert it back to this single line format ? Does the added spacing / lines affect the size of the project?

Reply:
You did the right thing. For some reason, some of the original source code for this extension was minified.

When we run `npx xt-build` it minifies it for us.

Reply:
good to know, thanks!